Unicode character in Encrypted Password Rule causes Sonar Json import to fail

Encrypted Password Rule has Unicode character U+2018 "‘" in both RULE_NAME and RULE_VIOLATION_MESSAGE strings.
When trying to import the resulting linter report into Sonarqube, the Sonarqube fails with an invalid JSON syntax error:

`
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:4.0.0.4121:sonar (default-cli) on project sample-mule-app: Failed to read external issues report 'C:\tmp\SampleMuleApp\reports\mule-linter-report-old.json': invalid JSON syntax: java.nio.charset.MalformedInputException: Input length = 1 -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:4.0.0.4121:sonar (default-cli) on project sample-mule-app: Failed to read external issues report 'C:\tmp\SampleMuleApp\reports\mule-linter-report-old.json': invalid JSON syntax
`

This was tested with Sonar plubing 4.0.0 and 3.9

--- a/mule-linter-core/src/main/groovy/com/avioconsulting/mule/linter/rule/property/EncryptedPasswordRule.groovy
+++ b/mule-linter-core/src/main/groovy/com/avioconsulting/mule/linter/rule/property/EncryptedPasswordRule.groovy
@@ -13,8 +13,8 @@ import com.avioconsulting.mule.linter.model.rule.RuleViolation
 class EncryptedPasswordRule extends Rule {
 
     static final String RULE_ID = 'ENCRYPTED_VALUE'
-    static final String RULE_NAME = 'The Property File contains ‘secret’ or ‘password’ values that are encrypted. '
-    static final String RULE_VIOLATION_MESSAGE = 'The Property file contains a ‘secret’ or ‘password’ that is not encrypted: '
+    static final String RULE_NAME = 'The Property File contains `secret` or `password` values that are encrypted. '
+    static final String RULE_VIOLATION_MESSAGE = 'The Property file contains a `secret` or `password` that is not encrypted: '
     static final String ENC_REGEX = '(\\!\\[.*?\\])'
 
     EncryptedPasswordRule() {

